config/deployment.yml

```diff
@@ -43,15 +43,17 @@ spec:
         volumeMounts:
         - name: template-fs
           mountPath: /etc/kappctrl-mem-tmp
-        securityContext:
-          runAsUser: 1000
-          runAsGroup: 2000
         ports:
         - containerPort: #@ data.values.api_port
           name: api
           protocol: TCP
-      securityContext:
-        fsGroup: 3000
```

Comment:
any doc links on how k8s picks fsGroup if not specified?

Reply:
No links but it appears that it'll become root (0)

Reply:
I think it'll be whatever fsGroup the volumes already were set to. I think `fsGroup` is applicable when you are mounting a volume with files that are not owned by your container's user, it basically does a `chmod/chown` on all the files/directories in the volume. We don't mount anything but emptyDir volumes so I think it's entirely irrelevant for our use-case.